rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -361,12 +377,23 @@ def test_upgrade_v4_v5(globaldb):
             nodes_tuples_from_db = cursor.execute('SELECT * FROM default_rpc_nodes').fetchall()
             assert nodes_tuples_from_db == nodes_tuples_from_file
 
-        last_queried_ts = globaldb_get_general_cache_last_queried_ts_by_key(
+        # check again existence of unique_cache table
+        cursor.execute(
+            'SELECT COUNT(*) FROM sqlite_master WHERE type="table" and name=?',
+            (UNIQUE_CACHE,),
+        )
+        assert cursor.fetchone()[0] == 1
+        value = globaldb_get_cache_values(
+            cursor=cursor,
+            key_parts=['TEST_KEY', '123'],
+        )
+        assert value == ['klm']
```

Comment:
This should be a unit tests in other place. This is testing that the upgrade works fine, not the logic in `globaldb_get_cache_values`